readme.md

```diff
@@ -43,14 +43,57 @@ npm install remark-language-server
 
 Usage of this package depends on your editor integration.
 Because this is based on
-[`vscode-languageserver-node`][vscode-languageserver-node], the same transports
+[`unified-languageserver-node`][unified-languageserver-node], the same features
 are supported.
 
-Practical examples are coming soon.
-
 `remark-language-server` uses the same configuration files as
 [`remark-cli`][remark-cli].
 
+## Examples
+
+### Atom
+
+The [`linter-remark`][linter-remark] plugin for Atom uses remark language server.
+It can be installed with:
+
+```sh
+apm install linter-remark
+```
+
+### Emacs
+
+Use [`lsp-mode`][lsp-mode] to use the remark language server with Emacs.
+
+### Neovim
+
+Use [`nvim-lspconfig`][nvim-lspconfig] to use the remark language server with
+Neovim.
+It’s enabled by default.
```

Comment:
````suggestion
The remark language server is part of [`nvim-lspconfig`][nvim-lspconfig], a collection of common configurations for Neovim's built-in language server client. This means you can add remark language server in one line:

```lua
require'lspconfig'.remark_ls.setup{}
```

Detailed installation instructions can be found [here](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#remark_ls)
````

Reply:
if it's possible to fallback to a global or bundled version of remark, can that be documented as well?

https://github.com/unifiedjs/unified-language-server/issues/31